examples/examples/subscriptions.rs

```diff
@@ -27,39 +29,63 @@ struct BookChanged {
     schema_path = "../schemas/books.graphql",
     graphql_type = "SubscriptionRoot"
 )]
+#[allow(dead_code)]
 struct BooksChangedSubscription {
     books: BookChanged,
 }
 
 #[async_std::main]
 async fn main() {
+    #[cfg(not(target_arch = "wasm32"))]
```

Comment:
Let's make multiple examples, there is no benefit to the reader in trying to put all examples in one file

Reply:
I agree, though one point in favor of having it combined is that this is what you currently need to do in order to create a cross-platform (wasm and native) app.

To start with, it's probably easier to split up as you say, then maybe think about examples and/or how to make using it cross-platform less ugly.

Reply:
I split it out into a examples-wasm folder, so the example Cargo.toml is less of a mess.